### ISSUE TITLE
Matrix room aliases are not tags

### ISSUE THREAD

**Issue by notramo** [writeas/writefreely#427]
Matrix room addresses start with a #, like #ipfs:matrix.org
When a Matrix room address is included in the text, its first part (the room name, without the server address) is incorrectly converted to a hashtag.
It should be converted to a https://matrix.to link, so when I write `#ipfs:matrix.org`, it should be converted to `[#ipfs:matrix.org](https://matrix.to/#/#ipfs:matrix.org)`.

**Comment by rkingett** [comment]
Maybe to fix this, we should have an edit box for tags and another edit box for categories beneath the post textarea.

**Comment by notramo** [comment]
No. If the tag is followed by a `:`, **then** it is a Matrix room alias, otherwise it is a tag.
So, the following text would resolve to a hashtag:
> You can download Arch Linux packages from #ipfs too, [more info here](https://github.com/RubenKelevra/pacman.store#readme)

But this would resolve to a Matrix room alias:
> You can ask questions regarding IPFS in #ipfs:matrix.org
